packages/builder/builder-doc/docs/en/config/security/checkSyntax.md

```diff
@@ -0,0 +1,28 @@
+- **Type**: `boolean | Syntax`
+
+```ts
+interface Syntax {
+  targets: string[];
+}
+```
+
+- **Default**: `false`
+
+Analyzes the generated product for the presence of high-level syntax that is not compatible in the specified environment. If present, the details are printed in the terminal.
+
+## Example
+
+```ts title="edenx.config.ts"
```

Comment:
edenx -> modern-js

Reply:
It should support more framework for builder, it should not bind with modern.js.

so delete the `titel=edenx(modern).config.ts`